test/integration/integration_test.go

```diff
@@ -316,6 +316,47 @@ func TestArchiveAndRead(t *testing.T) {
 	if retryErr != nil {
 		t.Fatal(retryErr)
 	}
+
+	// Query cronjob log urls
+	url = fmt.Sprintf("https://localhost:8081/apis/batch/v1/namespaces/%s/cronjobs/generate-log-1/log", namespaceName)
+	request, errRequest = http.NewRequest("GET", url, nil)
+	if errRequest != nil {
+		fmt.Printf("could not create a request, %s", errRequest)
+		t.Fatal(errRequest)
+	}
+	request.Header.Set("Authorization", fmt.Sprintf("Bearer %s", token.Status.Token))
+	fmt.Printf("HTTP request: %s\n", request.URL.RequestURI())
+
+	retryErr = retry.Do(func() error {
+		response, errResp := clientHTTP.Do(request)
+		if errResp != nil {
+			fmt.Printf("Couldn't get a response HTTP, %s\n", errResp)
+			return errResp
+		}
+		defer response.Body.Close()
+
+		body, errReadAll := io.ReadAll(response.Body)
+		if errReadAll != nil {
+			fmt.Printf("Couldn't read Body, %s\n", errReadAll)
+			return errReadAll
+		}
+
+		if response.StatusCode != http.StatusOK {
+			fmt.Printf("The HTTP status returned is not OK, %s\n", response.Status)
+			return errors.New(response.Status)
+		}
+		var logUrls []string
+		json.Unmarshal([]byte(body), &logUrls)
```

Comment:
Hey @maruiz93 I think that the `json.Unmarshal` could return an error that we should manage. Just like in line 305.

Reply:
Done!